### ISSUE TITLE
Typing `{#` causes weird autocomplete behaviour for logic blocks in VSCode

### ISSUE THREAD

**Issue by j-n-f** [sveltejs/language-tools#2274]
### Describe the bug

It seems like (maybe) some kind of VSCode autocomplete logic is interfering with autocompletion of logic blocks. I read through the documentation/issues and saw no mention of this.

### Reproduction

Svelte-related settings in settings.json:

```json
{
    "[svelte]": {
        "editor.defaultFormatter": "svelte.svelte-vscode"
    },
    "svelte.enable-ts-plugin": true
}
```

---

In an editor pane for a `.svelte` file, type

```
{#ea
```

to try and trigger autocompletion for `{#each`

1. When you type the opening bracket you get `{}`
2. When you type the hash you get `{#  #}` (the cursor is now centred in the string)
3. When you type `ea` you have `{# ea #}`

If I instead do `{@` I get `{@}` as an autocompletion (cursor placed after `@`).

If I do `{{` I get `{{  }}` as an autocompletion (with the cursor centred in the string).

### Expected behaviour

I would expect something more like:

1. Type `{`, see `{}`
2. Type `#`, see `{#}`
3. Type `ea`, see `{#ea}`, autocompletion hints would have a list containing `each`
4. Select autocompletion, see `{#each }` (with cursor positioned just before closing bracket)

### System Info

- OS: Windows 10
- IDE: VSCode 1.85.0, editing project files connected to WSL from windows version of VSCode
- Plugin Version: v108.2.1, running in WSL

### Which package is the issue about?

Svelte for VS Code extension

### Additional Information, eg. Screenshots

![image](https://github.com/sveltejs/language-tools/assets/154852324/d4c263b3-f3df-4107-850d-d577cf0859bb)

**Comment by jasonlyu123** [comment]
Can you share a screen recording? I don't quite understand what you mean. Also, can you test if another extension doesn't cause it? You can create an empty setting profile to test it so you can easily switch back afterwards.

**Comment by j-n-f** [comment]
I tried with a clean profile and couldn't reproduce it.

I have a lot of extensions, but was able to narrow it down to "Laravel Blade Spacer" (https://marketplace.visualstudio.com/items?itemName=austenc.laravel-blade-spacer). I guess `{# #}` is part of Laravel's blade template syntax, and the plugin decided to run on a `.svelte` file.

I also found that the Svelte LSP works fine in neovim, so I think we can close this ticket.
